modules/custom/az_paragraphs/src/Plugin/paragraphs/Behavior/AZTextWithMediaParagraphBehavior.php

```diff
@@ -147,6 +149,31 @@ public function buildBehaviorForm(ParagraphInterface $paragraph, array &$form, F
         ],
       ],
     ];
+    $form['title_level'] = [
+      '#title' => $this->t('Title heading level'),
+      '#type' => 'select',
+      '#options' => [
+        'h1' => $this->t('Page title (H1)'),
+        'h2' => $this->t('Section heading (H2)'),
+        'h3' => $this->t('Subsection heading (H3)'),
+        'h4' => $this->t('Subsection heading (H4)'),
+        'h5' => $this->t('Subsection heading (H5)'),
+        'h6' => $this->t('Subsection heading (H6)'),
+      ],
+      '#default_value' => $config['title_level'],
+      '#description' => $this->t('The heading level of the title. <a href="https://quickstart.arizona.edu/best-practices/using-headings" target="_blank">Learn about best web practices</a>.'),
+    ];
+    $form['title_alignment'] = [
+      '#title' => $this->t('Title alignment'),
+      '#type' => 'select',
+      '#options' => [
+        'text-left' => $this->t('Left'),
+        'text-center' => $this->t('Center'),
+        'text-right' => $this->t('Right'),
```

Comment:
```suggestion
        'text-left' => $this->t('Title left'),
        'text-center' => $this->t('Title center'),
        'text-right' => $this->t('Title right'),
```
To better distinguish this from content position with the same options.

Reply:
Should we also update the content position options to say `Content position left/center/right`? (Lines 97-99)

Reply:
@danahertzberg Yes, I think it would make sense to update the content position options as well (or have both sets of options as `Left`, `Center`, `Right`).

Would `Content left`, `Content center`, and `Content right` work? Or do you prefer keeping "position" in the option names?

Reply:
Per discussion in today's AZ Digital PR review, we plan to:
- Update the title options as indicated above
- Update the content position options to `Content left/center/right`
- Change "Content position" to "Content alignment" and update code accordingly